store/sqlstore/webhook_store.go

```diff
@@ -346,24 +346,22 @@ func (s SqlWebhookStore) AnalyticsIncomingCount(teamId string) (int64, *model.Ap
 	return v, nil
 }
 
-func (s SqlWebhookStore) AnalyticsOutgoingCount(teamId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		query :=
-			`SELECT 
+func (s SqlWebhookStore) AnalyticsOutgoingCount(teamId string) (int64, *model.AppError) {
+	query :=
+		`SELECT 
 			    COUNT(*)
```

Comment:
Here, the rest of the string must be have one tab less too to match with the previous indentation.